jdaviz/core/template_mixin.py

```diff
@@ -2315,7 +2319,9 @@ def _get_mark_coords_and_validate(self, viewer=None, selected=None):
         # (or selected_spatial_region) will fail.
         if np.any([len(obj) > 1 for obj in objs]):
             validity = {'is_aperture': False,
-                        'aperture_message': 'composite subsets are not supported',
+                        'aperture_message': 'composite subsets only support extraction '
+                                            'with aperture extraction method "center," and '
+                                            'without wavelength dependence',
```

Comment:
can we instead consider these as valid and add support for showing the orange outline?